### PR TITLE
Gaming event feature

### PR DESCRIPTION
**Pull request by mhnazary**
Create a web page with a mobile-first approach (responsive) and added the following features:

- [ ] Home page mobile/desktop version
- [ ] About page mobile/desktop version
- [ ] JS function for popup menu when click on the hamburger menu
- [ ] Fill out dynamically the Team section on the Home page with JS function
- [ ] Using grid and flex for the overall position of elements
- [ ] Interactive links to the home/about page when click it on the desktop navbar
- [ ] Linters fix

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+              <a href="#contact" id="modal__titles3"> News</a>
+            </div>
+        </div>
+        <i id="btn__open" class="fa fa-bars" aria-hidden="true"></i>
+      <nav class="socail_nav hide">
+        <div class="socail_nav_div">
+          <a href="#"
+            ><img src="./images/facebook.png" alt="facebook_icon"
+          /></a>
+          <a href="#"><img src="./images/twitter.png" alt="twitter_icon" /></a>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] Great design you got there. Please, as mentioned clearly in your project guidelines, this is someone's design(Cindy shin) you are suppose to personalize the content but not change the design, there fore Colors.
Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
,Layout: composition and space between elements should be followed. 
- please use a background almost similar to the gray color,
- your titles should follow the design color and weight, and they should be well positioned as well, as in the original design. Consider this as a means of proving to a client that you can replicate what he or she asks of you. 
- kindly implement this for mobile/desktop and the two pages. 

yours             |  original
:-------------------------:|:-------------------------:
Screenshot (1001)  |  Screenshot (1002)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        <li class="main_list_card">
+          <img
+            class="main_list_img"
+            src="./images/info_icon1.png"
+            alt="info1_pic"
+            srcset=""
+          />
+          <h3 class="main_list_header">Lecture</h3>
+          <span class="main_list_bio">
+            Lorem ipsum dolor sit, amet adipisicing elit.consectetur adipisicing
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] please kindly add the sizes(height) of the cards in the mobile version so they can be more visible and follow the original design as well.
Screenshot (999)_LI
original
Screenshot (1000)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+          </span>
+        </li>
+      </ul>
+      <button class="m_link">Join in this especial event!</button>
+      <a href="#" class="main_link">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+      <section class="team-page flex" id="Speakers">
+        <h2 class="subtitle">Meet the team</h2>
```
Comment (tufoinnkuo10) [inline index.html:142]: - [ ] please kindly add the line under the meet the team like in the original design.


yours             |  original
:-------------------------:|:-------------------------:
Screenshot (997)  |  Screenshot (998)

File: static/base_style.css [hunk 4]
```diff
@@ -0,0 +1,801 @@
...
+}
+
+.job-underline {
+  padding: 1px;
+  background: #272a31;
+  width: 1.8rem;
+}
+
+.job-position {
+  color: #ae2111b9;
```
Comment (tufoinnkuo10) [inline static/base_style.css]: - [ ] kindly use this  `#EC5242 `color here instead as used in the original design

File: README.md [hunk 5]
```diff
@@ -1 +1,205 @@
...
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="my_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (tufoinnkuo10) [inline README.md]: - [optional] great job removing the microverse logo and adding your. But it is not visible . You can remove it or fix it for a clean README file

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @mhnazary 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Design well followed

video presentation
- please always present yourself clearly(name, programmer) and the purpose of your project(why you built it and how you think it can be of help). also, try as much as possible for your face to be visible throughout the presentation. this is friendly. nice job

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Olamarx — APPROVED** [review]
Hi @mhnazary, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Readme file professional ✔️ 
- All required changes fixed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**